atlassian_jwt_auth/frameworks/common/asap.py

```diff
@@ -14,6 +14,13 @@ def _process_asap_token(request, backend, settings):
     if token is None and not settings.ASAP_REQUIRED and (
             settings.ASAP_REQUIRED is not None):
         return
+
+    if request and settings.ASAP_EXCLUDE_PATHS:
```

Comment:
If we want to do this I think this should be done in the various middleware classes or in a different function than this one.